scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -565,11 +565,11 @@ <h5 class="modal-title" id="archiveCaseLabel">Confirm archive case</h5>
     </div>
     <div>
       <a href="{{ url_for('cases.coverage_qc_report', institute_id=institute,
-                        case_name=case.display_name, date=date) }}" target="_blank"><i class="fa fa-link"></i></a>
+                        case_name=case.display_name) }}" target="_blank"><i class="fa fa-link"></i></a>
```

Comment:
Seems unused. May have been an idea there to append multiple date stamped ones, like for the delivery report.

Reply:
Perhaps!